pypeit/core/coadd1d.py

```diff
@@ -2077,6 +2078,9 @@ def combspec(waves, fluxes, ivars, masks, sn_smooth_npix,
               spectrum on wave_stack wavelength grid. True=Good.
     '''
 
+    waves = np.float64(waves)
```

Comment:
Okay this looks good. Maybe add a comment here that we cast to float64 because of a bug in np.histogram. Then I will approve. Thanks for tracking this down!!